receiver/statsdreceiver/README.md

```diff
@@ -19,12 +19,13 @@ Use case: it does not support horizontal pool of collectors. Desired work case i
 
 ## Configuration
 
-The following settings are required:
-
-- `endpoint` (default = `localhost:8125`): Address and port to listen on.
+The Following settings are optional:
```

Comment:
`endpoint` used to be under the required settings, right?

Has this changed? Maybe we can keep it there to avoid confusion with existing users unless it's wrong.